Riot/Views/RoomTitle/ExpandedRoomTitleView.xib

```diff
@@ -38,31 +38,31 @@
                     </constraints>
```

Comment:
I would prefer update `ExpandedRoomTitleView` class later in another PR.
Please remove your changes for the moment.